waspc/data/Generator/templates/server/src/auth/providers/email/signup.ts

```diff
@@ -163,3 +178,11 @@ function ensureValidArgs(args: object): void {
   ensurePasswordIsPresent(args)
   ensureValidPassword(args)
 }
+
+async function sendAccountAlreadyExistsEmail(email: string, content: Email): Promise<void> {
+  // Use the same email sending utility as verification emails, but do not update verification metadata.
+  // This avoids leaking info about verification status.
+  emailSender.send(content).catch((e) => {
+    console.error('Failed to send account already exists email', e);
+  });
+}
```

Comment:
Maybe we can use the same utility for email sending but adding an option for saving the metadata or not